text/atacseq-workflow.md

```diff
@@ -0,0 +1,168 @@
+    ```
+
+2. Split BAM file into multiple BAMs on the different read groups using `samtools split`. See [the samtools documentation](http://www.htslib.org/doc/samtools.html) for more information.
+
+    ```bash
+    samtools split -u $UNACCOUNTED_BAM_NAME \ # Reads that do not belong to a read group or the read group is unrecognized go here.
+                    -f '%*_%!.%.'              # Format of output BAM file names.
+    ```
+
+   If the BAM has unaccounted reads, those will need to be triaged and this step will need to be rerun.
+
+3. Run Picard `SamToFastq` on each of the BAMs generated in the previous step.
+
+    ```bash
+        picard SamToFastq \
+                INPUT=$INPUT_BAM \
+                FASTQ=$FASTQ_R1 \
+                SECOND_END_FASTQ=$FASTQ_R2 \
+                RE_REVERSE=true \
+                VALIDATION_STRINGENCY=SILENT
+    ```
+
+4. Run `fq lint` on each of the FastQ pairs that was generated in the previous step as a sanity check. You can see the checks that the `fq` tool performs [here](https://github.com/stjude/fqlib/blob/master/README.md#validators).
+
+    ```bash
+    fq lint $FASTQ_R1 $FASTQ_R2 # Files for read 1 and read 2.
+    ```
+
+5. Run `BWA-MEM`.
+
+    ```bash
+            bwa mem \
+                -t "$n_cores" \
+                -R $READ_GROUP \
+                $DB_PREFIX \
+                $fastq \
+                $fastq2 \
+                | samtools view -@ "$n_cores" -hb - \
+                > ~{output_bam}
+    ```
+
+6. Run `picard ValidateSamFile` on the aligned and sorted BAM file.
+
+    ```bash
+    picard ValidateSamFile \
+                    I=$CELL_RANGER_SORTED_BAM \     # Aligned, coordinate-sorted BAM.
+                    IGNORE=INVALID_PLATFORM_VALUE \ # Validations to ignore.
+                    IGNORE=MISSING_PLATFORM_VALUE
+    ```
+
+7. Find reads overlapping ENCODE exclude list sites
+
+    ```bash
+    bedtools intersect \                   # Report reads overlapping excluded intervals
+            -abam $BAM \
+            -b GRCh38_no_alt.exclude.bed.gz |
+        samtools view - | cut -f 1 >          # Retain only read names
+            $ENCODE_READS.txt
+        
+        java.sh org.stjude.compbio.sam.TweakSam -i $BAM
+            -u $ENCODE_READS.txt
+            -o $ENCODE_REMOVED.bam
+    ```
+
+8. Remove mitochondrial genome
+
+    ```bash
+    samtools view $ENCODE_REMOVED.bam chrM |
+        cut -f 1 > 
+        $MITO_READS.txt
+
+        java.sh org.stjude.compbio.sam.TweakSam -i $ENCODE_REMOVED.bam
+            -u $MITO_READS.txt
+            -o $MITO_REMOVED.bam
+    ```
+
+9. Remove supplemental alignments
+
+    ```bash
+    samtools view 
+        -f 2048 
+        $MITO_REMOVED.bam |
+        cut -f 1 >
+        $SUPPLEMENTAL_READS.txt
+
+        java.sh org.stjude.compbio.sam.TweakSam -i $MITO_REMOVED.bam
+            -u $SUPPLEMENTAL_READS.txt
+            -o $SUPPLEMENTAL_REMOVED.bam
+    ```
```

Comment:
What's the rational for removing supplemental alignments? My understanding is these flags are usually used to denote chimeric alignments. Is there a reason we want to remove all chimeric reads?